tests/installation/grub_set_bootargs.pm

```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use strict;
+use base "y2logsstep";
+use testapi;
+
+sub run() {
+    select_console('root-console');
+    script_run "source /etc/default/grub";
+    script_run 'new_cmdline=`echo $GRUB_CMDLINE_LINUX_DEFAULT | sed \'s/\(^\| \)quiet\($\| \)/ /\'`';
+    script_run 'sed -i "s#GRUB_CMDLINE_LINUX_DEFAULT=\"[^\"]*\"#GRUB_CMDLINE_LINUX_DEFAULT=\"$new_cmdline\"#" /etc/default/grub';
+    script_run 'grub2-mkconfig -o /boot/grub2/grub.cfg';
+    select_console('x11');
```

Comment:
how can you know it's the proper console? x11 doesn't exist in all the textmode installations ?